source/nodejs/adaptivecards-templating/typedoc.json

```diff
@@ -1,17 +1,16 @@
 {
 	"name": "Adaptive Cards Templating Javascript SDK",
-	"inputFiles": "./src",
-	"mode": "file",
+	"entryPoints": "./src/adaptivecards-templating.ts",
 	"out": "docs",
 	"exclude": [
-		"__tests__"
+		"**/__tests__/**"
 	],
-	"excludeNotExported": true,
 	"excludePrivate": true,
+	"excludeExternals": true,
 	"readme": "none",
 	"plugin": [
 		"typedoc-plugin-markdown"
 	],
 	"theme": "markdown",
-	"stripInternal": true
+	"excludeInternal": true
```

Comment:
Is it the new option which replaced stripinternal?

Reply:
I'm also interested in this topic, looks like stripInternal can modify the value of excludeInternal but some people have issues using stripInternal

http://typedoc.org/guides/options/#excludeinternal

https://github.com/TypeStrong/TypeDoc/issues/1469#:~:text=stripInternal%20is%20a%20TypeScript%20option%20%28https%3A%2F%2Fwww.typescriptlang.org%2Ftsconfig%23stripInternal%29.%20In%200.20%2C,for%20setting%20TypeScript%20options%20via%20TypeDoc%20config%20files.

Reply:
for what it's worth, we don't currently publish or otherwise consume these docs. I was just updating the configuration to continue working. we should take a more considered approach to API docs as part of our overall documentation effort (see #6316)